Gopkg.lock

```diff
@@ -172,10 +166,21 @@
   revision = "32fa128f234d041f196a9f3e0fea5ac9772c08e1"
 
 [[projects]]
-  name = "github.com/matttproud/golang_protobuf_extensions"
-  packages = ["pbutil"]
-  revision = "c12348ce28de40eed0136aa2b644d0ee0650e56c"
-  version = "v1.0.1"
+  branch = "master"
+  name = "github.com/operator-framework/operator-sdk"
```

Comment:
I think you need to run `dep ensure` again to update the lock file with the changes from the master.
You're missing the prometheus dependencies that got introduced in #323 
https://github.com/operator-framework/operator-sdk/blob/master/Gopkg.lock#L193-L199

Plus I don't know why the SDK is vendoring itself here. This should not be here. 
Can you remove your vendor directory and Gopkg.lock file and try `dep ensure` again.

I've tried your PR locally and it does not need to vendor the SDK.

Reply:
not sure either, will do, thanks @hasbro17